### PR TITLE
Merge in indexing updates

### PR DESCRIPTION
**Pull request by ericvolp12**

### CODE REVIEW

File: util/cliutil/util.go [hunk 1]
```diff
@@ -211,5 +218,15 @@ func SetupDatabase(dburl string, maxConnections int) (*gorm.DB, error) {
...
 
+	if isSqlite {
+		// Set pragmas for sqlite
+		if err := db.Exec("PRAGMA journal_mode=WAL;").Error; err != nil {
+			return nil, err
+		}
+		if err := db.Exec("PRAGMA synchronous=normal;").Error; err != nil {
+			return nil, err
+		}
+	}
```
Comment (bnewbold) [inline util/cliutil/util.go:229]: is this going to impact any other prod services? does Relay use sqlite?

File: search/bulk.go [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+
+	"github.com/ipfs/go-cid"
+)
+
+type pagerankJob struct {
+	did  syntax.DID
+	rank float64
+}
+
+// BulkIndexPageranks updates the pageranks for the DIDs in the Serch Index from a CSV file.
```
Comment (bnewbold) [inline search/bulk.go:26]: "Search" (nit)

File: search/query.go [hunk 3]
```diff
@@ -187,6 +187,25 @@ func (p *PostSearchParams) Filters() []map[string]interface{} {
 	return filters
 }
 
+// Filters turns search params in to actual elasticsearch/opensearch filter DSL
+func (p *ActorSearchParams) Filters() []map[string]interface{} {
```
Comment (bnewbold) [inline search/query.go:191]: ah, I forgot to add this, good catch

**Review by bnewbold — APPROVED** [review]
didn't go deep, but looks good!

File: util/cliutil/util.go (hunk 1 above)
Comment (ericvolp12) [reply util/cliutil/util.go:229]: Other things use sqlite (not relay) but they all want these pragmas anyhow, this is our standard practice for prod sqlite.

**Pull request merged**